File: problems/problem_4/fullboilerplate.java
import java.util.*;
public class Main {
    // {{USER_CODE}}
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.hasNext() ? sc.next() : "";
        if (!s.isEmpty()) {
            List<Character> res = separateCharacters(s);
            for (char c : res) {
                System.out.println(c);
            }
        }
    }
}

File: problems/problem_4/fullboilerplate.cpp
#include <bits/stdc++.h>
using namespace std;
// {{USER_CODE}}
int main() {
    string s;
    if (cin >> s) {
        vector<char> res = separateCharacters(s);
        for (char c : res) {
            cout << c << endl;
        }
    }
    return 0;
}